admin/server/telemetry.go

```diff
@@ -0,0 +1,37 @@
+package server
+
+import (
+	"context"
+
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"go.opentelemetry.io/otel/attribute"
+)
+
+func (s *Server) Telemetry(ctx context.Context, req *adminv1.TelemetryRequest) (*adminv1.TelemetryResponse, error) {
+	dims := make([]attribute.KeyValue, 0)
+	for k, v := range req.Event.AsMap() {
+		a, ok := toKeyValue(k, v)
+		if ok {
+			dims = append(dims, a)
+		}
+	}
+	s.uiActivity.Emit(ctx, "cloud-ui-telemetry", 1, dims...)
```

Comment:
What do you think about passing `name` and `value` as part of `TelemetryRequest` so that it will be easier to pass various metrics later?